.gitignore

```diff
@@ -33,3 +33,6 @@ xcuserdata
 /documentation
 test/fixtures/api/assets.zip
 test/fixtures/storage/assets.zip
+/.idea
+/cmake-build-debug
```

Comment:
Potential devs who use same toolchain might not know this and will find same problem.